### ISSUE TITLE
Fetch is not a function 

### ISSUE THREAD

**Issue by JJ-Kidd** [pusher/pusher-http-node#130]
We've started seeing the following error regarding fetch is not a function being returned to us since using version 4.0.1 of this package. This is happening when deployed to lambda after it has gone through webpack. I've found that changing 
`const fetch = require('fetch');` to `const fetch = require('fetch').default` in `lib/requests.js` it resolves this problem. If you could fix it or suggest alternatives that would be great.

Stack trace:
```
PusherFailureError: fetch is not a function
    at pushMessage (/Users/jordan/workspace/backend/services/test-api/.webpack/service/src/handler.js:317:11)
    at main (/Users/jordan/workspace/backend/services/test-api/.webpack/service/src/handler.js:12636:50)
    at InProcessRunner.run (/Users/jordan/workspace/backend/node_modules/serverless-offline/dist/lambda/handler-runner/in-process-runner/InProcessRunner.js:178:16)
    at async MessagePort.<anonymous> (/Users/jordan/workspace/backend/node_modules/serverless-offline/dist/lambda/handler-runner/worker-thread-runner/workerThreadHelper.js:29:14) {
  code: 'PUSHER_FAILURE'
```

**Comment by T-monius** [comment]
I'm having a similar error in a Lambda:

`TypeError: fetch is not a function
    at Object.send (/var/task/webpack:/tmp/seed/source/services/notes/node_modules/pusher/lib/requests.js:48:1)
    at Pusher.post (/var/task/webpack:/tmp/seed/source/services/notes/node_modules/pusher/lib/pusher.js:184:1)
    at Object.exports.trigger (/var/task/webpack:/tmp/seed/source/services/notes/node_modules/pusher/lib/events.js:56:1)
    at Pusher.trigger (/var/task/webpack:/tmp/seed/source/services/notes/node_modules/pusher/lib/pusher.js:153:1)
    at lambda (/var/task/webpack:/tmp/seed/source/services/notes/notePusher.js:15:10)
    at Runtime.handler (/var/task/webpack:/tmp/seed/source/services/notes/libs/handler-lib.js:12:20)
    at Runtime.handleOnce (/var/runtime/Runtime.js:66:25)`

**Comment by callum-oakley** [comment]
fixed in 4.0.2
